horovod/spark/torch/remote.py

```diff
@@ -56,8 +56,7 @@ def RemoteTrainer(estimator, metadata, last_checkpoint_state, run_id, dataset_id
     train_minibatch = train_minibatch_fn if train_minibatch_fn else _train_minibatch_fn()
 
     # If loss weight is not provided, use equal loss for all the labels
-    label_columns = estimator.getLabelCols()
-    loss_weights = estimator.getLossWeights()
+    loss_weights = to_list(estimator.getLossWeights())
```

Comment:
I don't think this will work.  If you look in https://github.com/horovod/horovod/blob/master/horovod/spark/torch/remote.py#L536 you'll see that we expect the length of loss_weights to always match the length of the labels.  We should so something similar to what we do on line 62 here.